stdlib/pathlib.pyi

```diff
@@ -236,6 +250,9 @@ class Path(PurePath):
     def replace(self, target: str | PurePath) -> Self: ...
     def resolve(self, strict: bool = False) -> Self: ...
     def rmdir(self) -> None: ...
+    if sys.version_info >= (3, 14):
+        def rmtree(self, ignore_errors: bool = False, on_error: Callable[[OSError], object] | None = None) -> None: ...
```

Comment:
I don't have this method on latest main branch of cpython.

```
$ ./python 
Python 3.14.0a0 (heads/main:5580f31c56e, Aug 10 2024, 22:50:58) [GCC 12.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pathlib
>>> pathlib.Path.rmtree
Traceback (most recent call last):
  File "<python-input-1>", line 1, in <module>
    pathlib.Path.rmtree
AttributeError: type object 'Path' has no attribute 'rmtree'
```

Reply:
It looks like they renamed it to `delete` since I tackled this.

Reply:
https://github.com/python/typeshed/pull/12487/commits/113f2508239a63eb2ed1e67579683ee324d0ab84